extensions/styledtext/libstyledtext.m

```diff
@@ -6,7 +6,7 @@
 
 #define get_objectFromUserdata(objType, L, idx) (objType *) * ((void **)luaL_checkudata(L, idx, USERDATA_TAG))
 
-// Lua treats strings (and therefore indexs within strings) as a sequence of bytes.  Objective-C's
+// Lua treats strings (and therefore indexes within strings) as a sequence of bytes.  Objective-C's
```

Comment:
this repositories uses `indices` and `indexes` (and has typos of both -- including in just this one file). I'm not enforcing a preference (and in fact, the changes for this repository were much more reliant on my faith in Google Sheets than normal -- I'm sorry about that -- I did correct a bunch of errors before making the PR, but things do slip through...).